lm_eval/tasks/mslr2022/utils.py

```
@@ -0,0 +1,208 @@
+import numpy as np
+import pandas as pd
+
+from datasets import Dataset, DatasetDict
+
+try:
+    import evaluate
+
+    evaluate.logging.set_verbosity(evaluate.logging.ERROR)
+    _bleu = evaluate.load("bleu")
+    _rouge = evaluate.load("rouge")
+except (ImportError, Exception):
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)